packages/components/src/DateTimePickers/DateRange/Picker/Picker.component.test.js

```diff
@@ -20,25 +20,19 @@ describe('DateRange.Picker', () => {
 		};
 
 		// when
-		const wrapper = mount(
+		const { container } = render(
 			<DateRangeContext.Provider value={managerValue}>
 				<Picker other="custom props" focusedInput="startDate" />
 			</DateRangeContext.Provider>,
 		);
+		// userEvent.click(screen.getByTestId('CalendarPicker'));
 
 		// then
-		expect(wrapper.find(CalendarPicker).at(0).props()).toMatchObject({
-			manageFocus: true,
-			onSubmit: managerValue.pickerManagement.onStartChange,
-			other: 'custom props',
-			selectedDate: new Date(2007, 0, 2),
-			endDate: new Date(2007, 1, 2),
-			t: expect.any(Function),
-			focusedInput: 'startDate',
-		});
+		expect(screen.getByLabelText('Date picker')).toBeVisible();
+		expect(container.firstChild).toMatchSnapshot();
```

Comment:
do we need snapshot testing ?
they have significant disadvantages, TMC is completely getting rid of snapshot testing

Reply:
snapshot is only good at one thing: keeping a state. I think this is still good to have one.
look at the test purpose: should render.
I don;t think you can have other purpose with snapshot. You can't write 
```javascript
it('should do this and that in the context of foo', () => {
  snapshot(render(<Component foo />))
}
```